### PR TITLE
007 user card

### PR DESCRIPTION
**Pull request by abhinavsanthosh2001**

### CODE REVIEW

File: 01-backend/springboot-library/src/main/java/com/project/springbootlobrary/responseModels/UserCard.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.project.springbootlobrary.responseModels;
+
```
Comment (rakesh-u09) [inline 01-backend/springboot-library/src/main/java/com/project/springbootlobrary/responseModels/UserCard.java]: remove extra spaces

**Comment by rakesh-u09** [comment]
Done

File: 01-backend/springboot-library/src/main/java/com/project/springbootlobrary/responseModels/UserCard.java (hunk 1 above)
Comment (abhinavsanthosh2001) [reply 01-backend/springboot-library/src/main/java/com/project/springbootlobrary/responseModels/UserCard.java]: Done

**Review by rakesh-u09 — APPROVED** [review]
Good

**Pull request merged**